src/components/ExpenseForm/ExpenseForm.jsx

```diff
@@ -1,23 +1,48 @@
 import React, { useState, useEffect } from 'react'
 import { DataService } from '../../services/DataService'
 import DatePicker from 'react-datepicker'
-import { expensesUrl } from '../../static/apiconfig'
+import ChooseCategory from './ChooseCategory/ChooseCategory'
```

Comment:
to avoid duplication of `ChooseCategory/ChooseCategory` I suggest to always create index.js with
```
import ChooseCategory from './ChooseCategory.jsx'
export default ChooseCategory
```
inside
Thanks to that you can use shorter import:
`import ChooseCategory from './ChooseCategory'`
If you like it please apply everywhere while we are at early stage of project.

Reply:
Done